packages/manager/apps/web/client/app/hosting/email/hosting-email.service.js

```diff
@@ -0,0 +1,93 @@
+import filter from 'lodash/filter';
+import isEmpty from 'lodash/isEmpty';
+
+import {
+  WEB_HOSTING_SERVICE_NAME,
+  WEB_HOSTING_EMAIL_FAMILY_NAME,
+  WEB_HOSTING_EMAIL_PRICING_MODE_DEFAULT,
+  WEB_HOSTING_CONF_ITEM_LEGACY_DOMAIN,
+} from './activate/activate.constants';
+
+export default class HostingEmailService {
+  /* @ngInject */
+  constructor($translate, OrderService, OvhApiHostingWeb, OvhApiOrder) {
+    this.$translate = $translate;
+    this.OrderService = OrderService;
+    this.OvhApiHostingWebEmailOption = OvhApiHostingWeb.EmailOption().v6();
+    this.OvhApiOrderCart = OvhApiOrder.Cart().v6();
+  }
+
+  getEmailOfferDetails(serviceName) {
+    return this.OvhApiHostingWebEmailOption.query({ serviceName }).$promise;
+  }
+
+  createCart(ovhSubsidiary) {
+    return this.OrderService.createNewCart(ovhSubsidiary);
+  }
+
+  assignCart(cart) {
+    return this.OrderService.assignCart(cart.cartId)
+      .then(() => cart);
+  }
+
+  getEmailServiceOptions(cart, serviceName) {
+    return this.OrderService.getProductServiceOptions(WEB_HOSTING_SERVICE_NAME, serviceName)
+      .then((options) => {
+        const option = filter(options, optn => optn.family === WEB_HOSTING_EMAIL_FAMILY_NAME
+            && !isEmpty(filter(
+              optn.prices,
+              price => price.pricingMode === WEB_HOSTING_EMAIL_PRICING_MODE_DEFAULT,
+            )));
+        return { cart, option: option[0] };
+      });
+  }
+
+  addServiceOption(cart, serviceName, option) {
+    const price = option.prices[0];
```

Comment:
what about other prices ?

Reply:
there is and will be only one price in array